client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/ExcelUploader/ContactUploader.tsx

```diff
@@ -18,16 +19,18 @@ const fileEndings = [
 ].map((suffix) => `.${suffix}`).join(',');
 
 interface ExcelUploaderProps {
-    contactEvent:number;
+    contactEvent: number;
     onSave: () => void;
+    allInteractions: Interaction[];
 }
-const ContactUploader = ({contactEvent, onSave}:ExcelUploaderProps) => {
+const ContactUploader = ({contactEvent, onSave, allInteractions} :ExcelUploaderProps) => {
+    const existingContacts = allInteractions.map(interaction => interaction.contacts).flat();
+    const existingContactsIds : string[] = existingContacts.map(contact => contact.idNumber).filter(id => id !== null && id !== '') as string[];
```

Comment:
check !id

Reply:
the right condition is id
i will change it